go-away/Cargo.toml

```
@@ -22,12 +27,12 @@ indoc = "1.0"
 go-away-derive = { path = "../go-away-derive", version = "0.5.0" }
 
 # Integration Deps
-uuid = { version = "0.8", optional = true }
-chrono = { version = "0.4", optional = true , features = ["serde"] }
+uuid = { version = ">=1.0.0, <2.0.0", optional = true }
```

I think this can be expressed as just:

```suggestion
uuid = { version = "1", optional = true }
```